### ISSUE TITLE
Request: Better accessibility support

### ISSUE THREAD

**Issue by andrewleader** [microsoft/AdaptiveCards#2024]
Narration, keyboard navigation

Also need to consider

- Use of proper HTML tags and attributes (e.g anchor element for hyperlinks, alt on image etc)
- Support for ARIA attributes
- Support for high contrast mode
- Support for supplying high contrast images
- backgroundColor on Image elements (letting authors supply high contrast colors)
- `Action.title` should be a tooltip for `selectActions` where appropriate
- Elements/Input and need an "ariaLabel" attribute that gets passed along to each renderer

Related details: 

* Tooltips on clipped TextBlock's (#1788)
* Hi-contrast remote images (#1648)

# Asks

* Action Center
